sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -3,12 +3,27 @@
 ## 7.0.0b6 (Unreleased)
 
 **New Features**
+
+* `renew_lock()` now returns the UTC datetime that the lock is set to expire at.
+* `receive_deferred_messages()` can now take a single sequence number as well as a list of sequence numbers.
 * Messages can now be sent twice in succession.
 
 **Breaking Changes**
 
+* Renamed `prefetch` to `prefetch_count`.
+* Renamed `ReceiveSettleMode` enum to `ReceiveMode`, and respectively the `mode` parameter to `receive_mode`
+* `retry_total`, `retry_backoff_factor` and `retry_backoff_max` are now defined at the `ServiceBusClient` level and inherited by senders and receivers created from it.
+* No longer export `NEXT_AVAILABLE` in `azure.servicebus` module.  A null `session_id` will suffice.
+* Renamed parameter `message_count` to `max_message_count` as fewer messages may be present for method `peek_messages()` and `receive_messages()` 
+* Renamed `PeekMessage` to `PeekedMessage`
+* Renamed `get_session_state()` and `set_session_state()` to `get_state()` and `set_state()` accordingly.
+* Renamed `session_id` to `id`
```

Comment:
I'm just thinking out loudly: I prefer `session_id` than `id` because it is consistent with the input `session_id` when creating the receiver.

but if we have got an agreement on renaming it to `id`, I won't bikeshed it anymore.

Reply:
This wasn't agreement so much as my assumption that because a user would be going "message.session.id" having "message.session.session_id" would be redundant.

Am fine to float this if you prefer otherwise.  Am about to be asking a list of questions to Johan so can tag this on.

Reply:
Johan was on your side of this, with the added ammo of avoiding id as a keyword.  I remain lukewarm on the "atm machine" like nomenclature but I follow the logic and will adjust this.